node/sign_js_only_package/action.yml

```diff
@@ -0,0 +1,91 @@
+name: Compress, Sign and Upload to GH Release
+description: 'Compresses package and signs with garasign and uploads to GH release'
+
+inputs: 
+  aws_role_arn:
+    description: 'AWS role input for drivers-github-tools/gpg-sign@v2'
+    required: true
+  aws_region_name:
+    description: 'AWS region name input for drivers-github-tools/gpg-sign@v2'
+    required: true
+  aws_secret_id:
+    description: 'AWS secret id input for drivers-github-tools/gpg-sign@v2'
+    required: true
+  npm_package_name:
+    description: 'The name for the npm package this repository represents'
+    required: true
+  dry_run:
+    description: 'Should we upload files to the release?'
+    required: false
+    default: 'true'
```

Comment:
Is it intentional that you _have_ to set dry_run to `false` to get this to work?

Reply:
Yes, it seemed safer to default to not uploading to avoid accidental uploads while modifying the scripts.  I think that's a sensible default to leave going forward too.

Reply:
Won't scripts already have dry_run set to false because they needed to do so? Where does the safety come in once the false settings are out there? 

dry run is opt-in with our other tools, it is surprising that it is reversed here